tests/virt_autotest/login_console.pm

```diff
@@ -121,7 +121,7 @@ sub login_to_console {
         }
     }
 
-    unless (is_tumbleweed or check_screen([qw(grub2 grub1 prague-pxe-menu)], 210)) {
+    unless (is_tumbleweed or check_screen([qw(grub2 grub1 prague-pxe-menu)], get_var('AUTOYAST') && !get_var("GUI_INSTALL") ? 1 : 210)) {
```

Comment:
Why add this
```
get_var('AUTOYAST') && !get_var("GUI_INSTALL") ? 1 : 210
```
?

Reply:
https://progress.opensuse.org/issues/128201#note-35

Reply:
I do not quite understand your change here. There is no need to change this part at all. And it seems that if you set GUI_INSTALL=0, then there will be actually no grub menu checking at all (because 1 second is not enough to capture anything and the system might not load grub menu at that time). I do not think this is acceptable. We use autoyast to install host which does not mean there is no need to detecting grub menu after install.